rpc_storage_handler.go

```diff
@@ -596,31 +596,8 @@ func (r *RPCStorageHandler) SetRollingWindow(keyName string, per int64, val stri
 
 }
 
-// SetScrollingWindow is used in the rate limiter to handle rate limits fairly.
 func (r *RPCStorageHandler) SetRollingWindowPipeline(keyName string, per int64, val string) (int, []interface{}) {
```

Comment:
So, if this function do not have any differences from `SetRollingWindow`, maybe make sense to remove it a all, and replace all references by `SetRollingWindow` ?